lib/app/routes/app_router.dart

```diff
@@ -647,13 +647,14 @@ class AppRouter {
               GenerosityChallengeHelper.isCompleted)) {
         if (state.user.needRegistration) {
           final createStripe = state.user.personalInfoRegistered;
-          context.pushNamed(
-            FamilyPages.registrationUS.name,
-            queryParameters: {
+          context
+            ..pushNamed(FamilyPages.registrationUS.name, queryParameters: {
               'email': state.user.email,
               'createStripe': createStripe.toString(),
-            },
-          );
+            })
```

Comment:
Why push both and not just one? :)
From the other page you can open the registration page

Reply:
The final design looks like a screen, so why would I open a screen from a screen?
I can open a bottomsheet though from the registration page, is that what you meant?

Reply:
I mean that you are doing 2x 
            ..pushNamed()
Right now.

But I don't understand why :)
You can only do a push to the new one, right?

Reply:
I am setting up the stack so that the GivtBackButton in the Appbar would go to the page under. I _can_ only push one, then overwrite the GivtBackButton to navigate instead of pop, but I am not sure what are the benefits of one or the other.

Reply:
I mean there already is onPressedForced so no hassle, but I would like to know why? 😅

Reply:
Anw its changed to only push 1 page 🤷‍♀️